.pre-commit/run_tests.sh

```diff
@@ -4,4 +4,4 @@
 MOD=$(go list -m)
 PKGS=$(echo "$@"| xargs -n1 dirname | sort -u | sed -e "s#^#${MOD}/#")
 
-go test $PKGS
+go test -timeout=5m $PKGS
```

Comment:
What is the problem you are trying to solve, since there is already a default timeout of 10m, so I doubt changing it to 5min is going to help...?

```
	-timeout d
	    If a test binary runs longer than duration d, panic.
	    If d is 0, the timeout is disabled.
	    The default is 10 minutes (10m).
```